app/helpers/application_helper.rb

```diff
@@ -246,8 +246,84 @@ def sanitize_and_decode(str)
   end
 
   def internal_config_label(method, content = nil)
-    content ||= method.to_s.humanize
-    content << "*" if method.in?(VerifySetupCompleted::MANDATORY_CONFIGS)
-    label_tag("site_config_#{method}", content)
+    content ||= "<span>#{method.to_s.humanize}</span>"
+    content << "<span class='site-config__required'>Required</span>" if method.to_sym.in?(VerifySetupCompleted::MANDATORY_CONFIGS)
+    label_tag("site_config_#{method}", raw(content),  class: 'site-config__label')
+  end
+
+  def site_config_description
```

Comment:
So currently we've hardcoded the description and placeholders in the view. I'm not sure if these can be stored with (rails-settings-cached) in the database or should we just have a hash that maps them. And if we do keep the hash, where does this function belong?

The reason we need it to be dynamic is because when we have a `Get Started` section we don't want to keep hardcoding the values but we want to simply use a loop to go through the mandatory configs.

Reply:
> I'm not sure if these can be stored with (rails-settings-cached) in the database or should we just have a hash that maps them.

Let's keep things simple for now, I think a lookup hash is fine for the time being.

> And if we do keep the hash that I've outlined below, does this function belong in the application helper or should it be moved somewhere else?

Maybe add it as `SiteConfigDescriptions` to `app/lib/constants.rb` to avoid recreating this fairly big hash on every method invocation.